gitleaves/reports.py

```diff
@@ -86,12 +103,17 @@ def gen_ghwiki_reports():
     next_leaves_by_month = ((calendar.month_name[month], leaves)
                             for month, leaves
                             in next_leaves_by_month_temp_jar.items())
+    export_month_csv(data)
+
     template = templates.get_template('ghwiki/Home.md')
     if not os.path.exists(ghwiki_reports_dir):
         os.makedirs(ghwiki_reports_dir)
     with open(f'{ghwiki_reports_dir}/Home.md', 'w') as report:
         report.write(template.render(today_leaves=today_leaves,
                                      next_leaves_by_month=next_leaves_by_month))
+
+    place_reports()
```

Comment:
@jason-at-scroll 
- for now drop this new function `place_reports` just call copyfile here 
- not a very strong opinion but open(dst/__Sidebar.md, 'w').write(src/_Sidebar.md) might be slightly better than importing shutils.copyfile

Reply:
Done!